index.html

```diff
@@ -157,42 +188,44 @@ <h4>Skills</h4>
       <li class="list">Terminal</li>
     </ul>
   </li>
+  <li class="dotsquare"></li>
 </ul>
+</section>
 
 <!-- Form section Starts -->
 <section class="form">
-  <section class="intro">
-    <a href="index.html"><img class="logo" src="images/1.png" alt="Logo"></a>
+  <section class="form_intro">
+    <!-- <img src="images/image-geometry_6.png" alt="Shape"> -->
+    <a href="index.html"><img class="logo2" src="images/1.png" alt="Logo"></a>
     <h5>Get Started</h5>
```

Comment:
- [ ] Get started section should have a lightly gray background, currently, on your design only form has the background: 

Your design  | Figma design
------------- | -------------
<img width="789" alt="Screenshot 2022-04-01 at 12 11 57" src="https://user-images.githubusercontent.com/77038610/161223026-2f54f66c-0cb9-43c2-af7a-ae9bd0e9c7a1.png"> | <img width="689" alt="Screenshot 2022-04-01 at 12 11 16" src="https://user-images.githubusercontent.com/77038610/161222909-f1be0628-6951-4159-a5ec-0817d15c4789.png">